liquidator/liquidator.js

```diff
@@ -15,6 +15,20 @@ class Liquidator {
 
     // Instance of the expiring multiparty to perform on-chain liquidations.
     this.empContract = this.empClient.emp;
+
+    // Set default config settings. Caller can override these settings by passing in new
+    // values via the `config` input object.
+    const defaultConfig = {
```

Comment:
open question: Is this the best place to put the `defaultConfig`? I only put it here because I wanted to access the same `web` that the bot is using.

Reply:
I think it's reasonable. If you want to make it a little more object oriented, you could write a separate function in the class that handles all of the defaults and config processing and call that in the mainline constructor. But I think either way works. IMO, having the config processing in the same place as the class is nice because it makes it a little more self-documenting.

Reply:
That's a good approach you suggested. We can move there pretty easily, I do like keeping the default values close to the constructor for documentation purposes though

Reply:
ye. I think this is a good spot for it. it is the default config for the liquidator, it should be in this implementation. Comments also help, good job with them.